config/watcher_local_test_gcm/kustomization.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namePrefix: klm-
+commonLabels:
+  app.kubernetes.io/instance: kcp-lifecycle-manager
+  app.kubernetes.io/name: lifecycle-manager
+  app.kubernetes.io/created-by: argo-cd
+  app.kubernetes.io/part-of: kcp
+  app.kubernetes.io/managed-by: kustomize
+images:
+  - name: europe-docker.pkg.dev/kyma-project/prod/lifecycle-manager
+generatorOptions:
+  disableNameSuffixHash: true
+resources:
+  - namespace.yaml
+  - ../manager
+  - crd/dns.gardener.cloud_dnsentries.yaml # needed for GCM to work since this is not included in the helm chart for GCM
```

Comment:
can we get this crd publicly in github repository? Then you can use remote address directly, it also allowing you to reference to a specific tag or commit https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#examples